src/fmt.rs

```diff
@@ -3,7 +3,7 @@
 //! TODO write example of usage
 use core::fmt::{Result, Write};
 
-impl<Word, Error> Write for ::serial::Write<Word, Error=Error>
+impl<Word, Error> Write for dyn (::serial::Write<Word, Error=Error>)
```

Comment:
Is it okay if `impl<Word, Error, T> Write for T where T: ::serial::Write<Word, Error=Error>` (take another proper name for T)?

Reply:
It's ok in new version of compillers - because it replace old trait implements.